ERROR - 2024-06-11 00:52:32 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'mamp'@'localhost' (using password: NO) in /Applications/MAMP/htdocs/zero/system/Database/MySQLi/Connection.php:186
Stack trace:
#0 /Applications/MAMP/htdocs/zero/system/Database/MySQLi/Connection.php(186): mysqli->real_connect('localhost', 'mamp', Object(SensitiveParameterValue), 'zero', 3306, '', 0)
#1 /Applications/MAMP/htdocs/zero/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 /Applications/MAMP/htdocs/zero/system/Database/BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#3 /Applications/MAMP/htdocs/zero/app/Controllers/Home.php(20): CodeIgniter\Database\BaseConnection->query('SELECT * FROM u...')
#4 /Applications/MAMP/htdocs/zero/system/CodeIgniter.php(933): App\Controllers\Home->teste()
#5 /Applications/MAMP/htdocs/zero/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 /Applications/MAMP/htdocs/zero/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /Applications/MAMP/htdocs/zero/system/Boot.php(312): CodeIgniter\CodeIgniter->run()
#8 /Applications/MAMP/htdocs/zero/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#9 /Applications/MAMP/htdocs/zero/public/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#10 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: NO) in /Applications/MAMP/htdocs/zero/system/Database/MySQLi/Connection.php:231
Stack trace:
#0 /Applications/MAMP/htdocs/zero/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 /Applications/MAMP/htdocs/zero/system/Database/BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
#2 /Applications/MAMP/htdocs/zero/app/Controllers/Home.php(20): CodeIgniter\Database\BaseConnection->query('SELECT * FROM u...')
#3 /Applications/MAMP/htdocs/zero/system/CodeIgniter.php(933): App\Controllers\Home->teste()
#4 /Applications/MAMP/htdocs/zero/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#5 /Applications/MAMP/htdocs/zero/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /Applications/MAMP/htdocs/zero/system/Boot.php(312): CodeIgniter\CodeIgniter->run()
#7 /Applications/MAMP/htdocs/zero/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#8 /Applications/MAMP/htdocs/zero/public/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#9 {main}
CRITICAL - 2024-06-11 00:52:32 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: NO)
[Method: GET, Route: teste]
in SYSTEMPATH/Database/BaseConnection.php on line 457.
 1 SYSTEMPATH/Database/BaseConnection.php(604): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH/Controllers/Home.php(20): CodeIgniter\Database\BaseConnection->query('SELECT * FROM usuarios')
 3 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Home->teste()
 4 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
 5 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH/Boot.php(312): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
